web/app/js/components/ResourceDetail.jsx

```diff
@@ -181,8 +183,20 @@ export class ResourceDetailBase extends React.Component {
           resourceIsMeshed = _get(this.state.resourceMetrics, '[0].pods.meshedPods') > 0;
         }
 
+        let isTcpOnly = false;
+        let resMetrics = resourceMetrics[0];
```

Comment:
ah, it does. Mostly I wanted a shorter name because we use `resourceMetrics[0]` a bunch here and I was finding it hard to read!

I think a better name would actually be `resourceMetric` or `metricForResource` but that's not actually significantly shorter :(

Maybe just `metric`?

Reply:
`metric` works for me, I think it's more clear!